backend/testfiles/execution/language/epipe.dark

```diff
@@ -41,6 +41,7 @@ type X = { y: Y }
 
 type MyEnum = A of Int64 * Int64 * Int64
 (33L |> MyEnum.A 21L 42L) = MyEnum.A 33L 21L 42L
```

Comment:
same here - not good. let's remove tests that don't align with our actual expectations